packages/wrangler/src/pages.tsx

```diff
@@ -1031,13 +1042,10 @@ const createDeployment: CommandModule<
       builtFunctions = readFileSync(outfile, "utf-8");
     }
 
-    type File = {
-      content: Buffer;
-      metadata: Metadata;
-    };
-
-    type Metadata = {
-      sizeInBytes: number;
+    type FileContainer = {
+      content: string;
+      contentType: string;
+      size: number;
```

Comment:
I quite liked having the unit of measurement easily visible (in the name `sizeInBytes`) since we have had problems before with things like time units being incorrect. If you don't want it in the name, then how about adding a JSDoc comment to this definition so it appears in the intellisense?